### PR TITLE
add initial height query to block fetcher (by querying genesis)

### PR DESCRIPTION
**Pull request by piercetrey-figure**
- bespoke, hand-coded updates to openapi file, as we don't know how this was generated still and it doesn't appear to match the reality of the genesis response produced from testnet

### CODE REVIEW

File: es-api-model/src/main/resources/tendermint-v0.34.12-rpc-openapi-FIXED.yaml [hunk 1]
```diff
@@ -1905,7 +1905,7 @@ components:
                 - "chain_id"
                 - "initial_height"
                 - "consensus_params"
-                - "validators"
+#                - "validators" # for some reason this schema has this as required, but it comes back as empty from the chain... maybe this is just out of date?
```
Comment (piercetrey-figure) [inline es-api-model/src/main/resources/tendermint-v0.34.12-rpc-openapi-FIXED.yaml:1908]: bespoke

File: es-api-model/src/main/resources/tendermint-v0.34.12-rpc-openapi-FIXED.yaml [hunk 2]
```diff
@@ -2938,11 +2938,19 @@ components:
         evidence:
           type: object
           required:
-            - "max_age"
+            - "max_age_num_blocks"
+            - "max_age_duration"
+            - "max_bytes"
           properties:
-            max_age:
+            max_age_num_blocks: # Manually updated these properties to reflect the current properties here in cosmos/tendermint
```
Comment (piercetrey-figure) [inline es-api-model/src/main/resources/tendermint-v0.34.12-rpc-openapi-FIXED.yaml:2945]: hand-coded

File: es-api-model/src/main/resources/tendermint-v0.34.12-rpc-openapi-FIXED.yaml [hunk 3]
```diff
@@ -2938,11 +2938,19 @@ components:
...
               type: string
               example: "100000"
+            max_age_duration:
```
Comment (piercetrey-figure) [inline es-api-model/src/main/resources/tendermint-v0.34.12-rpc-openapi-FIXED.yaml:2948]: organic

File: es-core/src/testFixtures/resources/templates/genesis/success.json
Comment (piercetrey-figure) [inline es-core/src/testFixtures/resources/templates/genesis/success.json:1]: fetched from testnet via
```bash
curl https://rpc.test.provenance.io:443/genesis | jq | pbcopy
```

File: es-core/src/testFixtures/resources/templates/genesis/success.json [hunk 4]
```diff
@@ -0,0 +1,758 @@
+{
+  "jsonrpc": "2.0",
+  "id": -1,
+  "result": {
+    "genesis": {
+      "genesis_time": "2021-03-09T18:45:22.152625998Z",
+      "chain_id": "pio-testnet-1",
+      "initial_height": "${initial_height:-1}",
```
Comment (piercetrey-figure) [inline es-core/src/testFixtures/resources/templates/genesis/success.json:8]: string substitution for test

**Review by hyperschwartz — APPROVED** [review]

**Pull request merged**